libmscore/score.cpp

```diff
@@ -4028,6 +4028,22 @@ ChordRest* Score::cmdNextPrevSystem(ChordRest* cr, bool next)
       return cr;
       }
 
+//---------------------------------------------------------
+//   cmdNextPrevFrame
+//   Return next/previous [Vertical/Horizontal/Text] frame
+//   to be used as a navigation command
+//---------------------------------------------------------
+
+Box* Score::cmdNextPrevFrame(MeasureBase* currentMeasureBase, bool next) const
+      {
+            Box* selectedBox { nullptr };
```

Comment:
I remember seeing a discussion about list initialization somewhere, but don't remember the outcome. Didn't we use `selectedBox = nullptr` in the rest of the codebase?

Reply:
Not sure what they used.